sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/LinkData.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace.data;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.sdk.trace.config.TraceConfig;
+
+/**
+ * Data representation of a Link.
+ *
+ * <p>Used (for example) in batching operations, where a single batch handler processes multiple
+ * requests from different traces. Link can be also used to reference spans from the same trace.
+ */
+public interface LinkData {
+
+  /**
+   * Returns a new immutable {@code Link}.
```

Comment:
```suggestion
   * Returns a new immutable {@link LinkData}.
```

If we change all these to `{@link}` than renames will be applied in the future (well hopefully we're GA and don't rename :)